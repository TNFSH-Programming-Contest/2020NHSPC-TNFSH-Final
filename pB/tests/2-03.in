76
-285.331 -391.626
-151.921 -377.41
-140.986 -266.965
-226.281 -238.533
-265.648 -287.742
-225.187 -321.641
-220.0 -282.0
-193.475 -299.77
-198.943 -339.137
-241.59 -350.072
-278.77 -319.454
-305.014 -277.9
-272.209 -225.411
-167.231 -205.728
-101.619 -253.842
-86.8 -348.6
-132.238 -426.619
-252.525 -452.863
-325.791 -438.648
-385.935 -352.259
-404.525 -248.375
-366.252 -128.087
-171.605 -120.433
-60.065 -169.641
-7.576 -282.0
-48.037 -461.612
-231.748 -509.727
-434.05 -479.108
-485.445 -304.144
-473.417 -224.317
-489.819 -137.929
-399.057 -130.275
-383.748 -71.224
-265.648 -65.757
-30.54 -86.534
93.028 -321.641
56.941 -420.058
94.121 -475.827
16.481 -531.597
-28.353 -521.755
-184.727 -584.086
-353.2 -548.4
-486.539 -588.46
-499.661 -457.237
-553.0 -282.0
-514.97 -26.39
-220.0 51.0
-466.856 -69.037
-512.783 -133.555
-520.438 -299.77
-461.388 -486.763
-476.697 -517.381
-419.8 -548.4
-352.036 -511.914
-263.461 -531.597
-109.274 -519.568
25.229 -478.014
25.229 -351.166
46.4 -282.0
-5.389 -201.353
-53.504 -123.713
-169.418 -89.814
-327.978 -98.562
-365.158 -90.908
-395.777 -164.174
-465.762 -160.893
-419.8 -215.4
-451.546 -315.08
-399.057 -448.489
-260.18 -482.388
-92.871 -447.396
-48.037 -356.633
-66.627 -229.785
-186.914 -155.426
-325.791 -190.418
-350.942 -321.641
-520.438 -299.77
