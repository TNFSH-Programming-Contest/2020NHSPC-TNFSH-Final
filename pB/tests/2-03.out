1283902.415